### PR TITLE
Use OGC server named QGIS server in the WFS permalink

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/vars.yaml [hunk 1]
```diff
@@ -74,6 +74,10 @@ vars:
           params:
             limit: 30
             partitionlimit: 5
+        ngeoPermalinkOgcserverUrl:
+          name: mapserverproxy
+          params:
+            ogcserver: 'QGIS server'
```
Comment (ochriste) [inline geoportal/vars.yaml:80]: do we only have qgis-server layers in the demo? or is this just set as an example ?
what happen if we want wfspermalink in a project using both mapserver and qgisserver ?

Comment (sbrunner) [reply geoportal/vars.yaml:80]: The example present in the test book uses layers from QGIS server!

Comment (sbrunner) [reply geoportal/vars.yaml:80]: It's not possible to create WFS permalink link one different OGC servers :-/

Comment (ochriste) [reply geoportal/vars.yaml:80]: ok, thank you

**Pull request merged**